src/ripple/app/misc/AmendmentTableImpl.cpp

```diff
@@ -50,7 +51,8 @@ class AmendmentTableImpl : public AmendmentTable
     core::Clock::time_point m_lastReport;  // close time of most recent majority report
     beast::Journal m_journal;
 
-    AmendmentState* getCreate (uint256 const& amendment, bool create);
+    AmendmentState& getCreate (uint256 const& amendment);
+    AmendmentState* getExisting (uint256 const& amendment);
```

Comment:
The problem is what "getExisting" should do if the element it's asking for isn't in the table. I don't to want getExisting to thow - it goes against how the rest of rippled uses exceptions. Not sure how to address this one.